Modules/@babylonjs/react-native-windows/package.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name": "@babylonjs/react-native-windows",
+  "title": "React Native Windows Babylon",
+  "version": "0.0.1",
+  "description": "Babylon Native integration into React Native Windows",
+  "main": "index.ts",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/babylonjs/BabylonReactNative.git",
+    "baseUrl": "https://github.com/babylonjs/BabylonReactNative"
+  },
+  "keywords": [
+    "react-native"
+  ],
+  "author": {
+    "name": "Your Name",
+    "email": "yourname@email.com"
+  },
+  "homepage": "https://github.com/BabylonJS/BabylonReactNative#readme",
+  "license": "MIT",
+  "licenseFilename": "LICENSE",
+  "readmeFilename": "README.md",
+  "peerDependencies": {
+    "@babylonjs/core": "^5.0.0-alpha.9",
```

Comment:
It seems like @babylonjs/react-native should be in this list, right?

Reply:
i am not sure how to add @babylonjs/react-native with the correct version given our current tooling. We could add it here as 0.0.1 so that package test evaluates correctly. I don't think this peerDependency version will be automatically changed by our npm packaging setup

Reply:
I was thinking we would manage it manually. The only part of @babylonjs/react-native that @babylonjs/react-native-windows cares about is the JavaScript right? So technically we'd only have to bump the min version when something in the JS <--> Native communication changes, right?

Does any of the native code (such as headers) get rebundled with @babylonjs/react-native-windows, or is it referenced from @babylonjs/react-native?

Reply:
This is just a thought though to make it a little more discoverable. We could certainly revisit this in a future PR.

Reply:
@babylonjs/react-native-windows resolves shared headers (BabylonNative.h) from @babylonjs/react-native

I don't think this suggestion of manually managing @babylonjs/react-native version dependencies will scale. It will break local packaging with the 0.0.1 versions. It will also then require prs to update the version prior to every release snap, which is going to be a new pain point for BabylonReactNative contributors

Reply:
i feel like there has to be some other mechanism for declaring a matching version for a dependency

Reply:
oddly enough, if i just add a "@babylonjs/react-native": "version" dependency, i don't see any complaints